include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -209,6 +210,10 @@ table KUKAiiwaArmConfiguration {
   /// position, cartesian impedence, or joint impedence low level controller adjustments
   controlMode:EControlMode;
 
+  setCartImpedance:CartesianImpedenceControlMode;
```

Comment:
Yes it looks like the following may be a reasonable way to set it up:

`Union ControlMode { PositionControlMode, CartesianImpedenceControlMode, JointImpedenceControlMode, NoControlMode} `